src/openrct2-ui/windows/Guest.cpp

```diff
@@ -49,32 +50,36 @@ enum WINDOW_GUEST_WIDGET_IDX {
     WIDX_TAB_4,
     WIDX_TAB_5,
     WIDX_TAB_6,
+    WIDX_TAB_7,
 
-    WIDX_MARQUEE = 10,
+    WIDX_MARQUEE = 11,
     WIDX_VIEWPORT,
     WIDX_ACTION_LBL,
     WIDX_PICKUP,
     WIDX_RENAME,
     WIDX_LOCATE,
     WIDX_TRACK,
 
-    WIDX_RIDE_SCROLL = 10
+    WIDX_RIDE_SCROLL = 11
 };
 
 validate_global_widx(WC_PEEP, WIDX_ACTION_LBL);
 validate_global_widx(WC_PEEP, WIDX_PICKUP);
 
+static const int32_t TabWidth = 30;
```

Comment:
This can be a `constexpr` as well.

Reply:
b4ea407dfbb7259ddf76b894d36cb8e64a169b23